### PR TITLE
Create 0000-01-02-Minahil006.md

### PR DESCRIPTION
**Pull request by Minahil006**
created a branch, 
created a file and made a commit, and opened a pull
request

### CODE REVIEW

File: e _posts/0000-01-02-Minahil006.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: “Welcome to our second slide!”
+---
+Your test
```
Comment (Minahil006) [inline e _posts/0000-01-02-Minahil006.md]: there should be something new

**Comment by Minahil006** [comment]
there should be someething new

File: e _posts/0000-01-02-Minahil006.md (hunk 1 above)
Comment (Minahil006) [reply e _posts/0000-01-02-Minahil006.md]: ok

**Review by Minahil006 — COMMENTED** [review]
ok

File: e _posts/0000-01-02-Minahil006.md (hunk 1 above)
Comment (Junaid006) [reply e _posts/0000-01-02-Minahil006.md]: Its good

**Review by Junaid006 — APPROVED** [review]
Its fine

**Review by Junaid006 — CHANGES_REQUESTED** [review]
Approved

**Review by Junaid006 — APPROVED** [review]
Approved

**Comment by Minahil006** [comment]
Ok Thanks for approving

**Comment by Minahil006** [comment]
Thanks

**Review by Junaid006 — APPROVED** [review]

**Review by Minahil006 — COMMENTED** [review]
ok

**Pull request merged**